### PR TITLE
[10.0] add module hr_timesheet_employee

### PR DESCRIPTION
**Pull request by yasmineelmrini**
the hr_timesheet_employee module adds an employee_id field to the account_analytic_line model, this field is altered only if the analytic line is a timesheet ; this module depends on the analytic_line_is_timesheet module which adds a new field called is_timesheet

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Why the user_id field isn't enough?

**Comment by ghost** [comment]
@pedrobaeza this module is very relevant when you have to build further logic around timesheet entries (I mean logic bound to the user). If you only think of timesheet entries as analytic accounting output, this field is not helpfull.
From one major release of Odoo to another, the concept of a timesheet line becomes always less defined.
One other reason is for reporting purposes.

**Comment by pedrobaeza** [comment]
Well, I obtain my statistics via user, and when I want to link with payrolls and so, I use the employee link in the user, but anyway, if you think it's important, go ahead. It was just curiosity.

File: hr_timesheet_employee/tests/__init__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
```
Comment (ghost) [inline hr_timesheet_employee/tests/__init__.py:20]: wrong licence here

**Pull request closed without merging**